src/app/app.html

```diff
@@ -1,14 +1,17 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title>
+          PVTrAck
```

Comment:
@petermarathas and I had a lot of trouble trying to get the right thing read at the right place.